integration_test/src/java/org/apache/heron/integration_topology_test/common/AbstractTestTopology.java

```diff
@@ -82,12 +102,30 @@ public final void submit() throws AlreadyAliveException, InvalidTopologyExceptio
   }
 
   public final void submit(Config userConf) throws AlreadyAliveException, InvalidTopologyException {
-    TopologyBuilder builder = new TopologyBuilder();
-
-    Config conf = buildConfig(new BasicConfig());
-    if (userConf != null) {
-      conf.putAll(userConf);
+    if (this.httpServerResultsUrl == null) {
```

Comment:
this `if ... else ...` clause shares several common code, let's reuse the duplicate part.

Reply:
Thanks! Fixed.